.github/workflows/overview-dag.yml

```
@@ -0,0 +1,123 @@
+          bash scripts/gh/overview-dag.sh docs/overview-dag.dot docs/overview-dag.svg
+      - name: Stage DAG for commit
+        if: github.event_name == 'pull_request'
+        run: |
+          cp -f docs/overview-dag.svg "$RUNNER_TEMP/overview-dag.svg"
+          cp -f docs/overview-dag.dot "$RUNNER_TEMP/overview-dag.dot"
+      - name: Verify DAG outputs
+        run: |
+          test -f docs/overview-dag.dot || { echo "docs/overview-dag.dot missing" >&2; exit 1; }
+          test -f docs/overview-dag.svg || { echo "docs/overview-dag.svg missing" >&2; exit 1; }
+      - name: Summarize DAG artifacts
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:2:18: Quote this to prevent word splitting

(shellcheck)